packages/lesswrong/components/editor/EditTitle.tsx

```diff
@@ -28,47 +30,53 @@ const styles = (theme: ThemeType): JssStyles => ({
   },
 })
 
-interface EditTitleProps extends WithStylesProps {
-  clearField: any,
-  document: any,
+const EditTitle = ({document, value, path, placeholder, updateCurrentValues, classes}: {
+  document: PostsBase,
   value: any,
   path: string,
   placeholder: string,
-}
+  updateCurrentValues: Function,
+  classes: ClassesType
+}) => {
+  const { flash } = useMessages()
+  const [lastSavedTitle, setLastSavedTitle] = useState<string>(document.title)
+  const {mutate: updatePost} = useUpdate({
+    collectionName: "Posts",
+    fragmentName: 'PostsMinimumInfo',
+  });
+  const { question } = document;
 
-class EditTitle extends Component<EditTitleProps,{}> {
-  UNSAFE_componentWillMount() {
-    const { addToSuccessForm } = this.context
-    const { clearField } = this.props
-    addToSuccessForm(() => clearField())
-  }
-  render() {
-    const { document, value, path, placeholder, classes } = this.props
-    const { question } = document;
+  const handleChangeTitle = useCallback((event) => {
```

Comment:
Hmm. I think my own experience is actually that I attempt to change my title, forget to click save, and then I _do_ get to keep all my edits in localhost but don't end up getting to keep the edits to my title, and then I have to redo my title a few times.

This doesn't seem super high stakes to me either way since people will actually be paying attention at the time they actually publish.

Is this something you feel strongly about resolving before we complete the merge?

Reply:
I'm not 100% sure how on what the preferred outcome here is, but created this issue for it.

https://github.com/ForumMagnum/ForumMagnum/issues/5645